lib/webhooks/deployment.payload.json

```diff
@@ -5,6 +5,7 @@
     "sha": "69a8b72e2d3d955075d47f03d902929dcaf74033",
     "ref": "master",
     "task": "deploy",
+    "name": "baxterthehacker/public-repo",
```

Comment:
> Whats the difference between the top level attributes and the inner objects?

During the preview period for the Deployments API, the payloads originally had those top-level attributes, but they were removed prior to shipping the final (production) version of the Deployments API. This [blog post](https://developer.github.com/changes/2014-10-21-deployment-webhook-payload-changes/) describes the changes.

Heaven's test fixtures are likely using the old payload format.

I've opened an internal issue for us to update these example payloads to reflect current production functionality. Thanks for helping us spot this discrepancy. :bow: